packages/ui/src/components/molecules/ReviewCard/styles.scss

```diff
@@ -0,0 +1,184 @@
+[data-fs-review-card] {
+  // --------------------------------------------------------
+  // Design Tokens for ReviewCard
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-review-card-border-color                                                 : var(--fs-border-color-light);
+  --fs-review-card-border-width                                                 : var(--fs-border-width);
+  --fs-review-card-padding-mobile                                               : 1.25rem 0; // 20px
+  --fs-review-card-gap-mobile                                                   : var(--fs-grid-gap-0);
+
+  // Header
+  --fs-review-card-header-width-desktop                                         : 7rem;  // 112px
+  --fs-review-card-header-gap                                                   : var(--fs-grid-gap-0);
+
+  // Date
+  --fs-review-card-date-color                                                   : var(--fs-color-text-light);
+  --fs-review-card-date-font-size                                               : var(--fs-text-size-0);
+  --fs-review-card-date-line-height                                             : var(--fs-text-size-base);
+  --fs-review-card-date-font-weight                                             : var(--fs-text-weight-regular);
+
+  // Text
+  --fs-review-card-text-gap                                                     : var(--fs-spacing-0);
+
+  // Text-Headline
```

Comment:
If you agree with this [suggestion](https://github.com/vtex/faststore/pull/2645/files#r1962709621) we can also update this tokens to use `title` instead of `headline`.